src/main/resources/json-schemas/productaanvraag/productaanvraag-dimpact.json

```diff
@@ -254,13 +254,13 @@
       },
       "betrokkenen": [
         {
-          "inpBsn": "999993896",
+          "inpBsn": "111222333",
```

Comment:
Why did you change this? Is `111222333` also available in the BRP mock? We use TEST_PERSON_HENDRIKA_JANSE_BSN everywhere in our itests.

Reply:
Just copied the new version of this file from github. It comes with 111222333.

This however does not seem to prevent integration tests to pass, so I guess it's not really tested atm

Reply:
Changed the BSN back to 999993896

Reply:
ok, cool! yeah, apparently we don't touch this particular spot in our itests then. might be good to add at some point but let's leave that for now I guess.